app/views/admin/deploy_groups/show.html.erb

```diff
@@ -40,7 +40,16 @@
                     class: "btn btn-default"
         %>
         <%= link_to "Deploy All Projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default", data: { method: "post" } %>
-      <% end %>
+        <%= link_to "Merge Elligable Stages",
```

Comment:
can we use the copied_from_id here and then do `merge 4 created stages` ? (4 = stages with copied_from_id)